app/scripts/proactive/model/Task.js

```diff
@@ -1,534 +1,535 @@
+							}
+						}
+					},
+					"Output Files": {
+						type: 'List',
+						itemType: 'Object',
+						fieldAttrs: {
+							'placeholder': 'outputFiles->files',
+							"data-help": 'Files produced by task that that will be transferred from computing nodes to your user or global spaces automatically.'
+						},
+						subSchema: {
+							"Excludes": {type: "Text", fieldAttrs: {'placeholder': '@attributes->excludes'}},
+							"Includes": {type: "Text", fieldAttrs: {'placeholder': '@attributes->includes'}},
+							"Access Mode": {
+								type: 'Select',
+								fieldAttrs: {'placeholder': '@attributes->accessMode'},
+								options: ["transferToUserSpace", "transferToGlobalSpace", "transferToOutputSpace", "none"]
+							}
+						}
+					},
+					// extra parameters for the simple view
+					"Host Name": {
+						type: "Text",
+						fieldAttrs: {
+							"data-help": 'A host name or a part of host name on which this task must be executed',
+							"simple-view": true
+						}
+					},
+					"Operating System": {
+						type: 'Select',
+						fieldAttrs: {
+							"data-help": 'An operating system name where this task must be executed.',
+							"simple-view": true
+						},
+						options: [
+						          {val: "any", label: "Any"},
+						          {val: "linux", label: "Linux"},
+						          {val: "windows", label: "Windows"},
+						          {val: "mac", label: "Mac"}
+						          ]
+					},
+					"Required amount of memory (in mb)": {
+						type: "Text",
+						fieldAttrs: {"data-help": 'A host with this amount of memory.', "simple-view": true}
+					},
+					"Dedicated Host": {
+						type: "Checkbox",
+						fieldAttrs: {
+							"data-help": 'Run the task exclusively on host - no other tasks will be executed in parallel on the same host.',
+							"simple-view": true
+						}
+					},
+					// Add the Fork Execution Environment select before the Fork Environment model. Because
+					// that is the only way to receive precise events. If something changes in the
+					// Fork Environment the whole model will be copied in the changed event, therefore specific events
+					// can't be distinguished. That is because Backbone js dos not support nested models (without
+					// appropriate plugins/frameworks). In this case, nested models are used but no code which
+					// handles them. That's why we arrived at this hybrid design.
+					"Fork Execution Environment": {
+						type: "Select",
+						options: ["User Defined", "Docker"],
+						fieldAttrs: {
+							// The Fork Execution Environment begins the Fork Environment tab, 'data-tab',
+							// everything which comes after this tab is included in it, if no new 'data-tab'
+							// is defined.
+							"data-tab": "Fork Environment",
+							"data-tab-help": "Fork environment is a new customisable JVM started to only run the task it belongs to. Also specify how to start this JVM, like in a Docker container for example.",

```

Comment:
Also specify how to start this JVM, like in a Docker container for example. 
? You do not miss something in this sentence?